src/lib/MUser.ts

```diff
@@ -735,20 +735,37 @@ GROUP BY data->>'clueID';`);
 		return this.caPoints() >= CombatAchievements[tier].rewardThreshold;
 	}
 
-	buildCATertiaryItemChanges() {
+	buildTertiaryItemChanges(
+		hasRingOfWealthI: boolean = false,
+		isInWilderness: boolean = false,
+		isOnSlayerTask: boolean = false
```

Comment:
Use a named object so that its obvious which is which when calling it, e.g. `buildTertiaryItemChanges({ hasRingOfWealthI:  true, inWildy: true, onTask: false });`